substrate/frame/staking/src/lib.rs

```diff
@@ -1144,15 +1113,18 @@ impl<T: Config> EraInfo<T> {
 		let overview = <ErasStakersOverview<T>>::get(&era, validator);
 
 		if overview.is_none() {
-			return ErasStakers::<T>::get(era, validator)
+			return Exposure::default()
 		}
 
 		let overview = overview.expect("checked above; qed");
 
 		let mut others = Vec::with_capacity(overview.nominator_count as usize);
 		for page in 0..overview.page_count {
-			let nominators = <ErasStakersPaged<T>>::get((era, validator, page));
-			others.append(&mut nominators.map(|n| n.others).defensive_unwrap_or_default());
```

Comment:
Note that you can convert bounded vec to vec with `into_inner`

Reply:
Ok, changed in https://github.com/paritytech/polkadot-sdk/pull/6445/commits/c881bb0015cfd0c9248ebd53f751ff267b5ba5d9